deployment/roles/haystack/tasks/main.yml

```diff
@@ -26,15 +26,14 @@
     group: "piaf-deployment"
     state: directory
     recurse: yes
-- name: "{{ client }} | copy init.py"
-  ansible.builtin.copy:
-    src: __init__.py
-    dest: "{{ client_installation_directory }}/haystack/__init__.py"
-    group: piaf-deployment
-- name: "{{ client }} | copy custom.py"
+- name: "{{ client }} | remove legacy custom file (2021-07-09, remove this after all clients have been installed)"
+  file:
+    path: "{{ client_installation_directory }}/haystack/custom.py" # this should be as same as `dest` above
+    state: absent
```

Comment:
this is to make sure the file is "absent" ?

Reply:
Yes exactly, it will delete the file if it exists.